estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models, fields
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'Property Offer'
+
+    price = fields.Float(required=True)
+    status = fields.Selection(
+        selection=[('accepted', 'Accepted'), ('refused', 'Refused'), ('pending', 'Pending')],
+        default='pending',
+    )
+
+    partner_id = fields.Many2one('res.partner', string="Partner")
+    property_id = fields.Many2one('estate.property', string="Property")
```

Comment:
```suggestion
    partner_id = fields.Many2one('res.partner', string="Partner", required=True)
    property_id = fields.Many2one('estate.property', string="Property", required=True)
```
Here both fields should be required, it does not make sense for an offer to exist w/o any of them